### PR TITLE
i1148: Add `model_run` to `burden_estimate` and `burden_estimate_stochastic`. 

### PR DESCRIPTION
**Pull request by MartinEden**
Either one of you to review.

https://vimc.myjetbrains.com/youtrack/issue/VIMC-1148

### CODE REVIEW

File: migrations/sql-annex/V2017.12.07.1405__AddModelRunToBurdenEstimateStochastic.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+ALTER TABLE burden_estimate_stochastic
+    /* Note lack of FK; this is deliberate */
+    ADD COLUMN model_run INTEGER NULL;
```
Comment (littledice) [inline migrations/sql-annex/V2017.12.07.1405__AddModelRunToBurdenEstimateStochastic.sql]: should this be nullable?

Comment (MartinEden) [reply migrations/sql-annex/V2017.12.07.1405__AddModelRunToBurdenEstimateStochastic.sql]: Good point, fixed.

**Comment by MartinEden** [comment]
How's this @richfitz @littledice?

**Comment by MartinEden** [comment]
Thanks, not sure why I missed that!

**Pull request merged**